src/main/steps/c100-rebuild/applicant/navigationController.ts

```diff
@@ -0,0 +1,62 @@
+import { Case } from '../../../app/case/case';
+import { C100Applicant, ChildrenDetails } from '../../../app/case/definition';
+import { applyParms } from '../../common/url-parser';
+import { C100_APPLICANT_ADD_APPLICANTS, C100_APPLICANT_RELATIONSHIP_TO_CHILD, PageLink } from '../../urls';
+
+class ApplicantDetailsNavigationController {
+  private applicantDetails: C100Applicant[] | [] = [];
+  private childrenDetails: ChildrenDetails[] | [] = [];
+  private childId: ChildrenDetails['id'] = '';
+  private applicantId: C100Applicant['id'] = '';
+
+  private getNextChild(): ChildrenDetails | null {
```

Comment:
can we reuse getNextChild and refactor getNextApplicant and getNextRespondent so that it can be reused by both applicant & respondent